sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/FormRecognizerAsyncClient.java

```diff
@@ -382,16 +382,23 @@ public FormRecognizerServiceVersion getServiceVersion() {
 
     private Function<PollingContext<OperationResult>, Mono<OperationResult>> receiptStreamActivationOperation(
         Flux<ByteBuffer> data, long length, FormContentType formContentType, boolean includeTextDetails) {
-        return (pollingContext) -> {
-            try {
-                Objects.requireNonNull(data, "'data' is required and cannot be null.");
-                Objects.requireNonNull(formContentType, "'formContentType' is required and cannot be null.");
-                return service.analyzeReceiptAsyncWithResponseAsync(ContentType.fromString(formContentType.toString()),
-                    data, length, includeTextDetails)
-                    .map(response -> new OperationResult(
-                        parseModelId(response.getDeserializedHeaders().getOperationLocation())));
-            } catch (RuntimeException ex) {
-                return monoError(logger, ex);
+        Objects.requireNonNull(data, "'data' is required and cannot be null.");
```

Comment:
This will throw an error that will cause the public async API to also throw an error. As per reactive guidelines, you should instead catch this exception in the public API method and return an error. See `PagedFlux` APIs for reference. You will have to do something similar for PollerFlux too.

File an issue for this and can be fixed post preview release.

Reply:
Thanks. Srikanta. #10408